setup.py

```diff
@@ -31,20 +31,27 @@
 from core import feconf # isort:skip
 
 
-# Configure the required packages and scripts to install.
-with open('requirements.txt', encoding='utf-8') as requirements_txt: # pylint: disable=replace-disallowed-function-calls
-    # The 'parse_requirements' returns a list of 'Requirement' objects.
-    # We need to transform these to strings using the str() function.
-    REQUIRED_PACKAGES = [
-        str(requirement)  # pylint: disable=replace-disallowed-function-calls
-        for requirement in pkg_resources.parse_requirements(requirements_txt)
-    ]
-
-setuptools.setup(
-    name='oppia-beam-job',
-    version=feconf.OPPIA_VERSION,
-    description='Oppia Apache Beam package',
-    install_requires=REQUIRED_PACKAGES,
-    packages=setuptools.find_packages(),
-    include_package_data=True,
-)
+def main() -> None:
+    """Builds python package used by Google Cloud Dataflow workers."""
+    # Configure the required packages and scripts to install.
+    with open('requirements.txt', encoding='utf-8') as requirements_txt: # pylint: disable=replace-disallowed-function-calls
+        # The 'parse_requirements' returns a list of 'Requirement' objects.
+        # We need to transform these to strings using the str() function.
+        required_packages = [
+            str(requirement)  # pylint: disable=replace-disallowed-function-calls
+            for requirement in pkg_resources.parse_requirements(
+                requirements_txt)
+        ]
```

Comment:
We were not able to import `setup.py` neither in the script nor on the python shell as it was not wrapped in a `main()` function. Adding a `main()` method won't affect its working since the script is executed from the shell itself and not from any other file.

Reply:
can you verify that the script still works using `python setup.py build` and the checking if _build/lib_ is generated?

Reply:
Sure. Here is the screencast:

https://user-images.githubusercontent.com/89375125/187223040-32acc017-edb3-4803-8fe0-cb3cee727f81.mp4